test/compress_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule CompressorTest do
+  use ExUnit.Case
+  doctest Compressor
+
+  test "compresses aaabccccdd to a3bc4d2" do
+    assert Compressor.compress("aaabccccdd") == "a3bc4d2"
+  end
+
+  test "aaaaaffffffffffc to a5f10c" do
+    assert Compressor.compress("aaaaaffffffffffc") == "a5f10c"
+  end
+
+  test "compresses abcd to abcd" do
+    assert Compressor.compress("abcd") == "abcd"
+  end
+
+  test "compresses ccceee12eccceee to c3e4c3e3" do
+    assert Compressor.compress("ccceee12eccceee") == "c3e4c3e3"
+  end
+
+  test "compresses effeac01cb65c to ef2eac2bc" do
+    assert Compressor.compress("effeac01cb65c") == "ef2eac2bc"
+  end
+
+  test "compresses mattisagoodprogrammer to mat2isago2dprogram2er" do
+    assert Compressor.compress("mattisagoodprogrammer") == "mat2isago2dprogram2er"
+  end
+
+  test "compresses empty string to empty string" do
+    assert Compressor.compress("") == ""
+  end
+
+  test "compresses 11123334 to empty string" do
+    assert Compressor.compress("11123334") == ""
+  end
+end
```

Comment:
Great job with committing the tests at the start and not needing to make any changes, other than the module name and adding the edge case. Pretty cool